### PR TITLE
Image search

### PR DESCRIPTION
**Pull request by davidpmccormick**
Relates to #4927

Uses a modal window for displaying items clicked on within image-only search. Based off [updated Zeplin designs](https://app.zeplin.io/project/5aab926b4b6efde01259e959/dashboard?tag=Modal).

__TODO:__
- [ ] The `useIsKeyboard` hook seems like it would be better suited to an e.g. `UtilsContext` – for state that is globally true for the app ('was the last interaction with the mouse or the keyboard') rather than true per-component. Our concept of `isEnhanced` should probably live there as well.
- [ ] The tooltip hasn't yet been designed, so has been left out of this PR.

### CODE REVIEW

File: catalogue/webapp/components/ExpandedImage/ExpandedImage.js [hunk 1]
```diff
@@ -1,128 +1,187 @@
...
+  ${props => props.theme.media.medium`
+    flex-basis: 60%;
   `}
-`;
 
-const LicenseWrapper = styled.div`
-  margin-bottom: 20px;
+  &:after {
+    position: absolute;
+    bottom: -1px;
```
Comment (gestchild) [inline catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: 🦄  ✨

Comment (davidpmccormick) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: 🎱 🔫

File: catalogue/webapp/components/ExpandedImage/ExpandedImage.js [hunk 2]
```diff
@@ -133,12 +192,25 @@ const ExpandedImage = ({ title, index, id, searchParams }: Props) => {
...
+      document.documentElement &&
+      document.documentElement.classList.add('is-scroll-locked');
+
+    return () => {
+      document &&
+        document.documentElement &&
+        document.documentElement.classList.remove('is-scroll-locked');
+    };
+  }, []);
+
```
Comment (gestchild) [inline catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: As several of the above useEffects all get called when the component mounts, could they be bundled together into a single useEffect ?

Comment (davidpmccormick) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: They could. [My take was that React encourages you to split different concerns into different `useEffect`s](https://reactjs.org/docs/hooks-effect.html#tip-use-multiple-effects-to-separate-concerns), which can improve readability and refactoring. I suppose there could be a perf penalty though.

Comment (gestchild) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: 👍

Comment (jamesgorrie) [reply catalogue/webapp/components/ExpandedImage/ExpandedImage.js]: Yea, the hooks implementation deals with this if you fancy digging into it.

**Pull request merged**